logstash.gemspec

```diff
@@ -2,84 +2,16 @@
 require File.expand_path('../lib/logstash/version', __FILE__)
 
 Gem::Specification.new do |gem|
-  gem.authors       = ["Jordan Sissel", "Pete Fritchman"]
-  gem.email         = ["jls@semicomplete.com", "petef@databits.net"]
-  gem.description   = %q{scalable log and event management (search, archive, pipeline)}
-  gem.summary       = %q{logstash - log and event management}
+  gem.authors       = ["Jordan Sissel", "Pete Fritchman", "Elasticsearch"]
+  gem.email         = ["jls@semicomplete.com", "petef@databits.net", "info@elasticsearch.com"]
+  gem.description   = %q{The core components of logstash, the scalable log and event management tool}
+  gem.summary       = %q{logstash-core - The core components of logstash}
   gem.homepage      = "http://logstash.net/"
   gem.license       = "Apache License (2.0)"
```

Comment:
Because otherwise old plugins does not work, they require a logstash.gemspec Not sure why, but I found that while changing everything. Do we like to have some kind of backwards compatibility?